nb2/service/person_service.py

```diff
@@ -31,6 +54,34 @@ def get_person_by_slack_user_id(slack_user_id: str):
     return Person.query.filter_by(slack_user_id=slack_user_id).one_or_none()
 
 
+def get_person_by_display_name(display_name: str):
+    """
+    Return Person with ghost_user_id if it exists.
```

Comment:
Should be updated to "Return Person with display_name if it exists"

Reply:
fixed!